ios/brave-ios/Sources/BraveNews/Composer/FeedDataSource.swift

```diff
@@ -947,7 +948,9 @@ public class FeedDataSource: ObservableObject {
         maxCount: 200,
         duplicateHandling: .removePerDay
       )
-      historyAPI.search(withQuery: "", options: options) { historyNodeList in
+
+      historySearchCancellable = historyAPI.search(withQuery: "", options: options) {
+        historyNodeList in
```

Comment:
Any reason not to use a local var like is used in shred?

Reply:
No specific reason. I can change it. I wasn't sure which to do here.